GOVERNANCE.md

```diff
@@ -51,23 +52,22 @@ If you're interested in reaching the next level and becoming a Maintainer, you c
 
 #### Nomination Process
 
-_Note: This process is still in progress, and the Discord bot that will power it is not yet built. For now, manually nominate/self-nominate by posting in Discord._
-
-- Self-nominate by running `!contribute` in the `#new-contributors` Discord channel and briefly describe your qualifying contribution (link recommended).
-- Connect your Discord account with GitHub (or Reddit, Twitter, etc.) to automatically get recognized for future contributions.
-
+- Self-nominate by sending `!contribute` in the Discord `#contribute` channel.
```

Comment:
I've never seen this documented anywhere, but I've heard a few people in Discord say that reaching "level 3" automatically confers the contributor role. (And, I know some people have been surprised to see themselves just become a different colour, so maybe that has happened, or maybe core have granted the role after noticing/having someone pointed out to them.

If these two other methods for reaching Contributor role are possible, then maybe they should be documented here, too? (Though, I get that these aren't "nominations" per se.) Just putting it out there.

Reply:
Yes, contribute once anywhere be a contributor everywhere!

See: https://github.com/withastro/astro/blob/5fde35196467f6e238989af9b8448b39098f0a81/GOVERNANCE.md#examples-of-recognized-contributions

Reply:
resolved in latest commit (ed81328)

Reply:
Discord is better than discord